packages/xdl/src/Simulator.js

```diff
@@ -153,6 +153,17 @@ export async function _isSimulatorRunningAsync() {
   }
 
   let bootedDevice = await _bootedSimulatorDeviceAsync();
+  
+  if (!bootedDevice) {
+    Logger.global.info(`Booting device in iOS simulator...`);
+    try {
+      return await _xcrunAsync(['simctl', 'boot', '5E560ABA-4019-4A8E-A1BB-F01C15533F0F']);
```

Comment:
Unfortunately this doesn't work for everyone, because the UDIDs of the devices aren't the same. For example, on my computer the UDID of the iPhone X simulator is `4BDC42E4-...` and there are no simulator devices with your UDID.

Looks like you can get the active / previously used device with this command:
```
defaults read com.apple.iphonesimulator CurrentDeviceUDID
```
However, I don't know if this will always be set. (What if simulator was never used before? etc). So perhaps have a fallback in case it's not set, e.g. find an iPhone that has `isAvailable: true` from the output of `xcrun simctl list --json`.